shared/html/index.html

```diff
@@ -28,7 +28,7 @@
   <!-- Default footer. -->
   <footer>
     <a target="_blank" href="http://groups.csail.mit.edu/haystack/">Haystack group</a> at <a target="_blank" href="http://csail.mit.edu/">MIT CSAIL</a> &amp; <a target="_blank"
-    href="http://bocoup.com">Bocoup</a> collaboration - <a target="_blank" href="http://github.com/haystack/tipsy">Source</a> / <a target="_blank" href="http://tipsy.csail.mit.edu">Project</a> - MIT License 2015
+    href="http://bocoup.com">Bocoup</a> collaboration – <a target="_blank" href="http://github.com/haystack/tipsy">Source</a> / <a target="_blank" href="http://tipsy.csail.mit.edu">Project</a> – MIT License 2017
```

Comment:
The year in the LICENSE file should have been 2014-2017 (based on the time when the file was first committed), and the year range here should be 2014-2017 instead.